baton/hub.go

```diff
@@ -40,45 +40,57 @@ func (h hub) run() {
 				close(c.send)
 			}
 		case rawMsg := <-h.receive:
-			fmt.Println(rawMsg)
 			var cmd command
 			if err := json.Unmarshal(rawMsg.data, &cmd); err != nil {
 				fmt.Println("nu")
 				log.Println(err)
 				break
 			}
-
+			fmt.Println(cmd)
 			msg := msg{rawMsg.conn, cmd}
+			processMessage(msg)
+		}
+	}
+}
 
-			fmt.Println(cmd)
-			resp := make(chan interface{})
-			module, ok := modules[msg.cmd.Module]
+func processMessage(message msg) {
+	resp := make(chan interface{})
+	module, ok := modules[message.cmd.Module]
+	if !ok {
+		fmt.Println(message.cmd.Module, "not in", modules)
```

Comment:
All of the issues about logging should be ignored until I begin on #12. I will change pretty much all of them then.